Documentation/ceph-common-issues.md

```diff
@@ -833,3 +834,10 @@ To overcome this, you need to increase the value of `fs.aio-max-nr` of your sysc
 You can do this with your favorite configuration management system.
 
 Alternatively, you can have a [DaemonSet](https://github.com/rook/rook/issues/6279#issuecomment-694390514) to apply the configuration for you on all your nodes.
+
+## Failed to create CRDs
+If you are using kubernetes version is v1.15 or older, you will see error like this
+```
+unable to recognize "STDIN": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1"
+```
+you need to create crd found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Since the `apiextensions.k8s.io/v1beta1` version of CRD is deprecated in k8s v1.16 and will no longer be supported fromv 1.19.
```

Comment:
```suggestion

you need to create CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Since the `apiextensions.k8s.io/v1beta1` version of CRD is deprecated in k8s v1.16 and will no longer be supported from v1.19.
```

Reply:
Done.